src/ui/map.js

```diff
@@ -36,7 +36,7 @@ define(function(require, exports, module) {
 
     this.configureMap = function(ev, config) {
       this.trigger('mapStarted', {});
-      // if list or facets are emabled, give the map less space
```

Comment:
yeah. i pondered this one for a while. what right do i have to cleanse history with my contemporary interpretation of English language?